### PR TITLE
CLDR-18850 Fix en, ckb, pap, etc.

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-18850

Fix en, ckb, pap, etc.

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: bin/.gitignore
Comment (srl295) [inline bin/.gitignore:1]: @macchiati please delete this file. doesn't belong here.

**Comment by srl295** [comment]
> Error: Key Validation Error: git-CLDR-18850-Fix-en,-ckb,-pap-f501ebb0baf8d67baf00071e715cbefd2e69da56 cannot contain commas.

@macchiati please don't use commas in branch names

File: bin/.gitignore
Comment (macchiati) [reply bin/.gitignore:1]: Appears to be gone now

**Comment by macchiati** [comment]
> > Error: Key Validation Error: git-CLDR-18850-Fix-en,-ckb,-pap-f501ebb0baf8d67baf00071e715cbefd2e69da56 cannot contain commas.
> 
> @macchiati please don't use commas in branch names

Does this have to be fixed before merge, or is it just for the future?

File: tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/locales/scriptToExemplars.txt [hunk 1]
```diff
@@ -25,9 +25,8 @@ Hans ;	292 ;	[丧乍乾仑仿伏佉佐佑佤侬俩俱倦偌傣储催兆兑净凉
...
 Hebr ;	12 ;	[פ{אַ}{אָ}{בֿ}{וּ}{יִ}{כּ}{פּ}{פֿ}{שׂ}{תּ}{ײַ}]
 Latn ;	369 ;	[a-mo-z·ß-öø-ÿāăąćĉċčďđēĕėęěĝğġģĥħĩīį-ıĵķĺļľłńņňŋōŏőœŕŗřśŝşšťŧũūŭůűųŵŷźżžƒƙơưƴǀ-ǃǎǐǒǔǖǘǚǜǝǥǧǩǫǯǵǹșțȟɓɔɖɗəɛɣɨ-ɪɬɲʉ-ʋʒʔʕʻʼˀḍḑḓḥḵḷḻḽḿṁṃṅṇṉṋṛṣṭṯṱṽẁẃẅẑẓạảấầẩẫậắằẳẵặẹẻẽếềểễệỉịọỏốồổỗộớờởỡợụủứừửữựỳỵỷỹꞌ{aː}{aˤ}{a̧}{a̱}{a᷆}{a᷇}{c̓}{eː}{e̱}{e᷆}{e᷇}{gʷ}{g̃}{g̱}{iː}{i̧}{i̱}{i᷆}{i᷇}{kʷ}{k̓}{k̓ʷ}{l̓}{m̀}{m̄}{m̓}{n̄}{n̓}{oː}{o̱}{o᷆}{o᷇}{p̌}{p̓}{qʷ}{q̓}{q̓ʷ}{t̆}{t̓}{uː}{u̧}{u᷆}{u᷇}{w̓}{xʷ}{x̌}{x̌ʷ}{y̓}{y̱}{àː}{áː}{áˤ}{ç̌}{èː}{éː}{ê̄}{ê̌}{ìː}{íj́}{íː}{òː}{óː}{ùː}{úː}{ą́}{ė̄}{ę́}{į́}{ŋ̀}{ŋ́}{ŋ̄}{ƛ̓}{ǝ̀}{ǝ́}{ǝ̂}{ǝ̄}{ǝ̌}{ǫ́}{ɔ̀}{ɔ́}{ɔ̂}{ɔ̃}{ɔ̄}{ɔ̈}{ɔ̌}{ɔ̧}{ɔ̱}{ɔ᷆}{ɔ᷇}{ə̀}{ə́}{ə̂}{ə̄}{ə̌}{ɛ̀}{ɛ́}{ɛ̂}{ɛ̃}{ɛ̄}{ɛ̈}{ɛ̌}{ɛ̧}{ɛ̱}{ɛ̱̈}{ɛ᷆}{ɛ᷇}{ɨ̀}{ɨ́}{ɨ̂}{ɨ̄}{ɨ̌}{ɩ̀}{ɩ́}{ɩ̂}{ɪ́}{ʉ̀}{ʉ́}{ʉ̂}{ʉ̄}{ʉ̈}{ʉ̌}{ʊ̀}{ʊ́}{ʊ̂}{ʕʷ}{ʕ̓}{ʕ̓ʷ}{ẹ̀}{ẹ́}{ọ̀}{ọ́}]
-Mymr ;	63 ;	[က-ဃစ-ဏဒ-နဖ-ဘဟ-ဧဩ-ာဲံ့္ှဿၵ-ၸၺ-ႄႇ-ႊ​ꧣ-ꧥ]
+Mymr ;	58 ;	[က-ဃစ-ဏဒ-နဖ-ဘဟ-ဢိ-ေး်-ွၵ-ၸၺ-ႄႇ-ႊ​ꧣ-ꧥ]
```
Comment (srl295) [inline tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/locales/scriptToExemplars.txt:28]: we need to just get rid of this test, https://unicode-org.atlassian.net/browse/CLDR-18384
